app/src/main/java/id/rizmaulana/covid19/ui/adapter/DetailAdapter.kt

```diff
@@ -17,7 +17,8 @@ import id.rizmaulana.covid19.util.ext.visible
  */
 class DetailAdapter(
```

Comment:
We will use `visitable pattern` or `adapter delegate` instead later this will `class` become deprecated

Reply:
yes i think thats ok for this PR, thats why i asked you to update yours in the previous thread

Reply:
Okayy 👍